test/e2e/common/node/pods.go

```diff
@@ -224,23 +212,8 @@ var _ = SIGDescribe("Pods", func() {
 		ginkgo.By("creating the pod")
 		name := "pod-submit-remove-" + string(uuid.NewUUID())
 		value := strconv.Itoa(time.Now().Nanosecond())
-		pod := &v1.Pod{
```

Comment:
this would be easier to review if it simply wrapped with the MustMixinRestrictedPod call and left the rest of the pod alone

Reply:
Ack. I've been thinking about adding a set of "PodBuilder" utilities for e2e tests. Maybe I should just defer the RestrictedPod stuff to that...

In a few of these cases, the image was changed from nginx. I spent a while trying to get nginx to run as non-root, but it really needs image changes to make it work, and these tests don't actually care what image is being run.